guides/common/modules/con_metrics-overview.adoc

```diff
@@ -1,19 +1,18 @@
 [id='metrics-overview_{context}']
 = Metrics Overview
 
-Obtaining metrics from {Project} is useful for troubleshooting a current issue, and capacity planning.
-This guide describes how to collect live metrics and archive them for a fixed period of time.
+You can set up a solution to collect live metrics from {ProjectServer}, archive them for a fixed period of time, and analyze them.
```

Comment:
What does "solution" mean? Is there a more descriptive word? Or maybe "... set up a third party solution to ..."? or "external solution"?